.github/workflows/publish-new.yml

```diff
@@ -1,8 +1,23 @@
-name: Publish new version of public packages
+name: Publish new version of public packages from main HEAD
 # This bumps the version, updates the changelogs, publishes a GitHub release, and publishes the packages to npm.
 
 # Package publishing is manually triggered on github actions dashboard
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      bump_type:
+        type: choice
+        description: Version Bump Type
+        required: true
+        default: 'minor'
+        options:
+          - minor
+          - major
+          - override
```

Comment:
Are you sure about "override" vs "patch"? Also, maybe minor -> regular or cadence?

Reply:
Yeah it's not for patch bumps. It's for using a custom override version number.